components/UI/Drawer/components/types.ts

```diff
@@ -4,7 +4,15 @@ export type Channel =
   | "ahbap"
   | "generic"
   | "teleteyit"
-  | "uydu";
+  | "uydu"
+  | "hospital"
+  | "satellite"
+  | "food"
+  | "sahra_mutfak"
+  | "pharmacy"
+  | "safe"
+  | "deprem_ihtiyac";
+
 export type ExtraParams = BabalaParameters | TwitterParameters;
```

Comment:
we will have a API for values of this

Reply:
keeping all of these like these is seems not ok. we can fetch these keys from a single source.